authors/settings.py

```diff
@@ -138,6 +140,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+        #'authors.apps.authentication.backends.JWTAuthentication',
```

Comment:
It was commented out in the PR from Sulaiman. I ended up with two commits but I've since brushed up on my version control skills. His commented it out because JWT is not yet implemented and that line produces lots of errors because it expects the implementation to be present.

Reply:
ok